docs/modules/boot.md

```diff
@@ -0,0 +1,2 @@
+# boot
+The boot module is responsible for booting the system. It also launches other processes and sysmodules. Boot is integrated with the pm (process manager) sysmodule in Atmosphère.
```

Comment:
As with the comment earlier, `boot` does not actually launch other processes. That's `boot2`'s job. Boot2 is the one that's embedded in pm.